CONTRIBUTORS.md

```diff
@@ -92,5 +92,5 @@ Thiago Chacon | MSD | 1.0, 1.1, 2.3
 Tiago Tresoldi | CTLG | 1.1, 1.2, 2.0, 2.1, 2.2, 2.3, 2.4
 Timotheus A. Bodt | CTP | 2.4
 Viola Kirchhoff da Cruz | CLS | 1.0 
-Wang Feng | D | 1.0
+Wang Feng | D | 1.0Yue Chen | D | 
```

Comment:
There is a line break missing.
Yue Chen is your HiWi, right? I would add "C" for "typed off and corrected concept lists".
And you can also add yourself with "LG". 
In the column `VERSION', you can add "2.4".

Reply:
okay, fixed. plus added to Hiwis that I forgot initially